time.md

```diff
@@ -1086,7 +1086,7 @@ A very handy way of scheduling a function call after a specific duration. It is
 
 Some of my favourites are
 
-- `time.After(duration)` which return you a `chan Time` when the duration has expired. So if you wish to do something _after_ a specific time, this can help. 
```

Comment:
The main fix is return to returns, but then I figured I'd make it consistent with the second bullet's wording.